### PR TITLE
Use riff-system as the namespace for kafka.

### PR DESCRIPTION
**Pull request by ericbottard**
This aligns with what we use for helm charts and makes the topics script work again

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -50,7 +50,7 @@ make dockerize
 
 [source, bash]
 ----
-kubectl apply -f config/kafka
+kubectl apply -n riff-system -f config/kafka
```
Comment (trisberg) [inline README.adoc]: this depends on the namespace being there - we should add instructions to create it like we do https://github.com/projectriff/riff/blob/master/Getting-Started.adoc#install-kafka-chart[here]

Comment (ericbottard) [reply README.adoc]: Can you add that one line, or do you want me to do it and merge?

Comment (trisberg) [reply README.adoc]: Sure, I can add it on merge

Comment (trisberg) [reply README.adoc]: Also adding `-n riff-system` to the teardown script

**Comment by trisberg** [comment]
Rebased and merged as commit 6717f5af8908591c9ab344976204cc19fa633c3e

**Pull request closed without merging**